app/models/risk_area.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class RiskArea < ApplicationRecord
+  self.inheritance_column = nil
+
+  enum type: {
+    flooding: 0,
+    theft: 1,
+    harassment: 2
```

Comment:
```suggestion
    harassment: 2,
    other: 3
```

It would be interesting to leave an enum value for 'other', we will never be able to list all the possibilities and occasions.